backend/templates/includes/navbar.html

```diff
@@ -41,4 +43,11 @@
             </ul>
         </div>
     </div>
+    <!-- <img src="../includes/nav_logo.svg" height='50'/>  -->
```

Comment:
It's good practice to delete dead/commented out code if it's no longer useful, just to keep the files clean.